packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx

```diff
@@ -205,6 +197,13 @@ export const LiveblocksPlugin = ({
         const provider = new LiveblocksYjsProvider(room, doc);
         yjsDocMap.set(id, doc);
         providersMap.set(id, provider);
+
+        provider.on("sync", () =>
+          providerEventSource.notify({
+            id: room.id,
+            status: getEditorStatus(provider),
+          })
+        );
```

Comment:
You may have to call `provider.off("sync")` to unsub, if I remember correctly. Not sure if "returning" the unsub function works with the Yjs Provider API.